js/src/carousel.js

```diff
@@ -523,6 +552,19 @@ class Carousel extends BaseComponent {
   }
 
   // Static
+  // Boosted mod: add pause button
+  static PauseCarousel(event) {
+    const pauseButton = event.target
+    const pauseButtonAttribute = pauseButton.getAttribute(SELECTOR_CAROUSEL_TO_PAUSE)
+    const carouselToPause = Carousel.getOrCreateInstance(document.querySelector(pauseButtonAttribute))
+    if (pauseButton.classList.contains('pause')) {
```

Comment:
Could this condition be replaced by `carouselToPause._isPaused` ?

Reply:
No because `carouselToPause._isPaused` will never be true here.

The goal here is to set the carousel to `pause after` a click on the pause button and to set it to `cycle` after a click on the play button.